db/migrate/20191210122304_create_products.rb

```diff
@@ -0,0 +1,11 @@
+class CreateProducts < ActiveRecord::Migration[6.0]
+  def change
+    create_table :products do |t|
+      t.string :name
+      t.string :desc
```

Comment:
you should go with `description` Clarity is good.

Reply:
Ok